Xamarin.Forms.Core/CarouselView/CarouselView.cs

```diff
@@ -0,0 +1,133 @@
+
+		public Color IndicatorsTintColor
+		{
+			get { return (Color)GetValue(IndicatorsTintColorProperty); }
+			set { SetValue(IndicatorsTintColorProperty, value); }
+		}
+
+		public static readonly BindableProperty CurrentPageIndicatorTintColorProperty = BindableProperty.Create("CurrentPageIndicatorTintColor", typeof(Color), typeof(CarouselView), Color.Gray);
+
+		public Color CurrentPageIndicatorTintColor
+		{
+			get { return (Color)GetValue(CurrentPageIndicatorTintColorProperty); }
+			set { SetValue(CurrentPageIndicatorTintColorProperty, value); }
+		}
+
+		public static readonly BindableProperty IndicatorsShapeProperty = BindableProperty.Create("IndicatorsShape", typeof(IndicatorsShape), typeof(CarouselView), IndicatorsShape.Circle);
+
+		public IndicatorsShape IndicatorsShape
+		{
+			get { return (IndicatorsShape)GetValue(IndicatorsShapeProperty); }
+			set { SetValue(IndicatorsShapeProperty, value); }
+		}
+
+		public static readonly BindableProperty ShowIndicatorsProperty = BindableProperty.Create("ShowIndicators", typeof(bool), typeof(CarouselView), false);
+
+		public bool ShowIndicators
+		{
+			get { return (bool)GetValue(ShowIndicatorsProperty); }
+			set { SetValue(ShowIndicatorsProperty, value); }
+		}
+
+		public static readonly BindableProperty ItemsSourceProperty = BindableProperty.Create("ItemsSource", typeof(IEnumerable), typeof(CarouselView), null);
+
+		public IEnumerable ItemsSource
+		{
+			get { return (IEnumerable)GetValue(ItemsSourceProperty); }
+			set { SetValue(ItemsSourceProperty, value); }
+		}
+
+		public static readonly BindableProperty ItemTemplateProperty = BindableProperty.Create("ItemTemplate", typeof(DataTemplate), typeof(CarouselView), null);
+
+		public DataTemplate ItemTemplate
+		{
+			get { return (DataTemplate)GetValue(ItemTemplateProperty); }
+			set { SetValue(ItemTemplateProperty, value); }
+		}
+
+		public static readonly BindableProperty PositionProperty = BindableProperty.Create("Position", typeof(int), typeof(CarouselView), 0, BindingMode.TwoWay);
+
+		public int Position
+		{
+			get { return (int)GetValue(PositionProperty); }
+			set { SetValue(PositionProperty, value); }
+		}
+
+		public static readonly BindableProperty AnimateTransitionProperty = BindableProperty.Create("AnimateTransition", typeof(bool), typeof(CarouselView), true);
+
+		public bool AnimateTransition
+		{
+			get { return (bool)GetValue(AnimateTransitionProperty); }
+			set { SetValue(AnimateTransitionProperty, value); }
+		}
+
+		// UWP only
+		public static readonly BindableProperty ShowArrowsProperty = BindableProperty.Create("ShowArrows", typeof(bool), typeof(CarouselView), false);
+
+		public bool ShowArrows
+		{
+			get { return (bool)GetValue(ShowArrowsProperty); }
+			set { SetValue(ShowArrowsProperty, value); }
+		}
+
+		public EventHandler<int> PositionSelected;
```

Comment:
How about provide sub-class of EventArgs instead of int to passing value?

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/event
✓ DO use System.EventArgs or its subclass as the type of the second parameter of the event handler, and call it e. 

Or How about do not passing value(int)?
Because it can get by itself